qcodes/__init__.py

```diff
@@ -115,4 +115,8 @@ def test(**kwargs):
     return retcode
 
 
-test.__test__ = False # type: ignore # Don't try to run this method as a test
+test.__test__ = False  # type: ignore # Don't try to run this method as a test
+
+from ._version import get_versions
```

Comment:
why was it needed for duplicate code? isn't `from . import version \n __version__ = version.__version__` enough?

Reply:
Sure